files/mark.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "mark.js",
+  "repo": "julmot/mark.js",
+  "files": {
+    "basePath": "dist/"

```

Comment:
Please let them for consistency as I'm offering them in all other distribution processes.